test/e2e/serial/workload_placement_test.go

```diff
@@ -1657,6 +1667,231 @@ var _ = Describe("[serial][disruptive][scheduler] workload placement", func() {
 				return ok
 			}, time.Minute, time.Second*5).Should(BeTrue(), "resources not restored on %q", targetedNodeName)
 		})
+
+		It("[test_id:47674][reboot_required][slow] should be able to modify the configurable values under the NUMAResourcesOperator and NUMAResourcesScheduler CR", func() {
```

Comment:
please also add a label to signal this test depends on some images, like `[images]`